src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/resolver/UserIdArgumentResolver.java

```diff
@@ -38,10 +40,15 @@ public Object resolveArgument(
 
         String userId = null;
         //Inside UserIdArgumentResolver
-        UserInfo userInfo = jwtGrantedAuthoritiesConverter.getUserInfo();
+        UserInfo userInfo = idamRepository.getUserInfo(getUserToken());

```

Comment:
(As above)
The code suggests that loading the userInfo from IDAM is required only to get the unique ID of the user - this information is already present in the userToken (JWT). A simpler solution is to parse the JWT and get the `uid` claim from there.